plugins/cratedb/provisioner.go

```diff
@@ -0,0 +1,37 @@
+package cratedb
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+var argsToProvision = []string{
+	"--username", fieldname.Username,
+	"--hosts", fieldname.Host,
+}
+
+type crateArgsProvisioner struct {
+}
+
+func crateProvisioner() sdk.Provisioner {
+	return crateArgsProvisioner{}
+}
+
+func (p crateArgsProvisioner) Provision(ctx context.Context, in sdk.ProvisionInput, out *sdk.ProvisionOutput) {
+	if value, ok := in.ItemFields[fieldname.Password]; ok {
+		out.AddEnvVar("CRATEPW", value)
+	}
+	
+	out.CommandLine = append(out.CommandLine[1:], argsToProvision...)
```

Comment:
I believe this would provision something like `--username username --host host` instead of the actual values for the username and host. 

This PR offers an example on how to achieve your desired goal: https://github.com/1Password/shell-plugins/pull/276/files#diff-df810172d90d1487b505f35899626347713d972eb51ae4d47818da6478f39fadR27

But feel free to schedule a pair programming session with us if you'd like to have a look together, this is a rather complex provisioning mechanism 😄

Reply:
sorry i didn't notice it